src/mapping.conf

```diff
@@ -67,9 +67,11 @@
         "battery.voltage"       :   "voltage.battery",
 
         "outlet.realpower"      :   "realpower.default",
+        "outlet.power"          :   "power.default",
```

Comment:
Beware that outlet.power is due to deprecation!
You should better use "input.power". The same is true for "input.realpower".

Ref https://github.com/networkupstools/nut/blob/master/drivers/eaton-mib.c#L573